proto/penumbra/penumbra/core/asset/v1/asset.proto

```diff
@@ -86,8 +87,36 @@ message Value {
 message ValueView {
   // A value whose asset ID is known and has metadata.
   message KnownAssetId {
+    // The amount of the value.
     core.num.v1.Amount amount = 1;
+
+    // The asset metadata describing the asset of the value.
     Metadata metadata = 2;
+
+    // Optionally, a list of equivalent values in other numeraires.
+    // 
+    // For instance, this can provide a USD-equivalent value relative to a
+    // stablecoin, or an amount of the staking token, etc.  A view server can
+    // optionally include this information to assist a frontend in displaying
+    // information about the value in a user-friendly way.
+    repeated EquivalentValue equivalent_values = 3;
+    message EquivalentValue {
+      // The equivalent amount of the parent Value in terms of the numeraire.
+      core.num.v1.Amount equivalent_amount = 1;
+      // Metadata describing the numeraire.
+      Metadata numeraire = 2;
+    }
```

Comment:
Does this mean:
A) every item in this list is equal to the original (aka many options)
or
B) when summed, the list in total is equal in value to the original (aka parts of a whole)

It appears that it's option A, however, in the case of an LP NFT, option B is more helpful in communicating the equivalent value

Reply:
It means (A).

For the LPNFT case, we probably don't just want to have reserves but also information about the position state, price, etc. So rather than try to shoehorn that into a "price" mechanism I think it's better to put the Position state into an Any.